### DISCUSSION TITLE
Repository license not detected

### DISCUSSION THREAD

**Discussion by mccroweyclinton-EPA** [community/community#53739] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

For some reason the license on my repository, [USEPA/RAQSAPI](https://github.com/USEPA/RAQSAPI) is no longer being detected even though I have a license file.

Can some one please help me with this?

**Comment by airtower-luna** [comment]
I see "Unknown, MIT licenses found", which seems accurate: You have a `LICENSE.md` with the MIT license, and a `LICENSE` which contains a duplicate of the copyright notice but doesn't match any well-known license. I'd expect removing the latter to solve the problem.
